Kitodo/src/test/java/org/kitodo/production/services/command/CommandServiceTest.java

```diff
@@ -158,7 +158,7 @@ public void runNotExistingScriptAsync() throws InterruptedException {
         String commandString = scriptPath + "not_existing_script" + scriptExtension;
         CommandService service = new CommandService();
         service.runCommandAsync(commandString);
-        Thread.sleep(1000); // wait for async thread to finish;
+        Thread.sleep(2000); // wait for async thread to finish;
```

Comment:
This change and similar changes increase the test execution time for everyone :-( and the test execution itself is already really long.

Reply:
We don't need to discuss the one second. This is just a single test being run. Yes, on a slower laptop, 1 second sometimes doesn't work and the build aborts.

Reply:
One second at one point, an other second an other point. Yes, this is not much at a single place but in common this could be extend the execution time. But for now I'm fine with this change.